pages/events/index.vue

```diff
@@ -86,42 +86,45 @@
           </div>
         </template>
         <template slot="extra-info">
-          <div class="event-page__section">
-            <h3>Follow our event calendar</h3>
-            <p class="event-page__section__description">
-              <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
-              Stay up to date with all of our scheduled events by following our calendar. You can view the calendar by visiting <AppLink v-bind="qiskitCalendarLink">{{ qiskitCalendarLink.label }}</AppLink>, or subscribe to it by adding to the calendar app of your choice.
-            </p>
-            <div class="event-page__tabs">
-              <cv-tabs aria-label="Calendar applications">
-                <cv-tab
-                  v-for="{ name, instructions } in calendarsInstructions"
-                  :id="`tab-${name}`"
-                  :key="name"
-                  :label="name"
-                  class="event-page__tab"
-                >
-                  <p class="event-page__sync">
-                    Start by copying the calendar subscription link
-                    <cv-code-snippet
-                      kind="inline"
-                      feedback-aria-label="Copy calendar sync url"
-                    >
-                      <span>{{ qiskitCalendarSyncLink }}</span>
-                    </cv-code-snippet>
-                  </p>
-                  <component :is="instructions" class="event-page__instructions" />
-                </cv-tab>
-              </cv-tabs>
+          <div id="follow-our-event-calendar" class="event-page__anchor">
+            <a href="#follow-our-event-calendar" />
+            <div class="event-page__section">
+              <h3>Follow our event calendar</h3>
+              <p class="event-page__section__description">
```

Comment:
It's weird to have the `id` and the `<a>` separate from the target `h3`, could you try if it works with this code?

```suggestion
         <div class="event-page__section">
            <a href="#follow-our-event-calendar" class="event-page__anchor"/>
            <h3 id="follow-our-event-calendar">Follow our event calendar</h3>
            <p class="event-page__section__description">
```